client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -531,6 +532,20 @@ export const confirmPackages = ( orderId, siteId ) => ( dispatch, getState ) =>
 	tryGetLabelRates( orderId, siteId, dispatch, getState );
 };
 
+export const saveCustoms = ( orderId, siteId ) => {
+	return {
+		type: WOOCOMMERCE_SERVICES_SHIPPING_LABEL_SAVE_CUSTOMS,
+		siteId,
+		orderId,
+	};
+};
+
+export const confirmCustoms = ( orderId, siteId ) => ( dispatch, getState ) => {
```

Comment:
What's the different between saving customs and confirming customs?

Reply:
`confirmCustoms` is the action performed as a result of the user clicking the "Confirm" button. It closes the field, fires the `saveCustoms` action to trigger a full validation, and tryes to fetch rates if the step was valid.

I copied the action names and meanings from the `Packages` step. See `confirmPackages` and `savePackages`. Any suggestions to make them more readable?

Reply:
Cool. Not yet - I'll see how the next PR goes. I do see that it's the same naming as packages, which is great.